pkg/server/metrics/metrics.go

```diff
@@ -22,11 +22,12 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-type Direction string
-
 const (
 	namespace = "konnectivity_network_proxy"
 	subsystem = "server"
+
+	Frontend = "frontend"
+	Backend = "backend"
```

Comment:
How about using the actual method names (connect/proxy)?

Frontend/Backend can be confusing.

Reply:
Good idea, done.

Do you think it's worth adding service name as well, for completeness? (We only have one method per service though.)

Reply:
I think the chance of us adding new methods of the same name is low, so "connect"/"proxy" are probably good enough.